### PR TITLE
Creating a template for people while making a PR 

### PR DESCRIPTION
**Pull request by harshil15999**
1.A PR template ensures consistent and clear communication of changes, streamlining the review process and enhancing code quality.
2.It standardizes information, reduces errors, and fosters better collaboration and accountability within the team.
3.Will reduce the cognitive burden on maintainer to review the PR

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+[A brief overview of the feature being added. Explain the functionality and its intended purpose.]
+
+## Implementation Details
+[Describe your approach to implementing the feature. Include any design decisions and methods used.]
+
+## Impact of the Feature
+[Discuss how this feature affects the existing codebase, including any potential behavioural or performance changes.]
+
+## Testing Conducted
+[Detail the tests carried out to ensure the feature works as intended and integrates well with the existing system.]
```
Comment (amandesai01) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: I think this many questions are overkill. Just keep one field called description.
PLUS, we need a field to link issue, that is super mandatory.

**Review by amandesai01 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (harshil15999) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: Ill keep feature ,implementation and testing

Comment (harshil15999) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: It will help in faster PR reviews

Comment (amandesai01) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: For now, let's go with simplicity. We can update if we find challenges later on.

**Pull request merged**